modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -102,13 +102,14 @@ dynamicPlugins:
 dynamicPlugins:
   frontend:
     default.main-menu-items: # key for configuring static main menu items
```

Comment:
I would expect we do not use the dottet notation in the YAML examples. Can you check with SME and QE?

```suggestion
    default:
    	main-menu-items: # key for configuring static main menu items
```